### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by YUXUANGE-111** [aclap-dev/video-downloadhelper#1191] [Help and Q&A]
### Your question

I paid for your product on Tue27 Aug2024,  because I thought the software was paid for and actually I didn't need the content of the paid features. I am very sorry. I hope to get a refund. There are no technical problems, your software is very good, but I misunderstood that I do not need the membership function. My top ten license key is: 3c1fbcba73

### Addon & CoApp version and details

I paid for your product on Tue27 Aug2024,  because I thought the software was paid for and actually I didn't need the content of the paid features. I am very sorry. I hope to get a refund. There are no technical problems, your software is very good, but I misunderstood that I do not need the membership function. My top ten license key is: 3c1fbcba73

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
